src/linelast_solver.cpp

```diff
@@ -179,17 +174,23 @@ void LinElastSolver::SetupRHSBCOperators()
          if (idx < 0)
             continue;
 
-         if (type_idx[b] == LinElastProblem::BoundaryType::NEUMANN)
-         {
-            bs[m]->AddBdrFaceIntegrator(new VectorBoundaryLFIntegrator(*bdr_coeffs[b]), *bdr_markers[b]);
-         }  
-
          if (!BCExistsOnBdr(b))
             continue;
 
-         if (type_idx[b] == LinElastProblem::BoundaryType::DIRICHLET) // Default behavior
-         { 
-            bs[m]->AddBdrFaceIntegrator(new DGElasticityDirichletLFIntegrator(*bdr_coeffs[b], *lambda_c[m], *mu_c[m], alpha, kappa), *bdr_markers[b]);
+         switch (bdr_type[b])
+         {
+            case BoundaryType::DIRICHLET:
+               bs[m]->AddBdrFaceIntegrator(new DGElasticityDirichletLFIntegrator(
+                  *bdr_coeffs[b], *lambda_c[m], *mu_c[m], alpha, kappa), *bdr_markers[b]);
+               break;
+            case BoundaryType::NEUMANN:
```

Comment:
Here we get a null function passed to the integrator with the current zero Neumann